components/form-control/script.js

```diff
@@ -65,6 +67,76 @@ class GamefaceFormControl extends HTMLElement {
         return elements;
     }
 
+    /**
+     * Will add an element where the errors about the form element with the passed name will be displayed
+     * @param {string} name - The name attribute value of a form element
+     * @param {string} selector - The selector of the element where the errors will be displayed
+     */
+    setCustomDisplayErrorElement(name, selector) {
+        if (!name || !selector) return;
+        if (typeof selector !== 'string') return;
+
+        const element = document.querySelector(selector);
+        if (!element) return;
+
+        this.errorDisplayElements[name] = element;
+    }
+
+    /**
+     * Method for removing element where the errors related to the form element with the passed name should be displayed
+     * @param {string} name - The name attribute value of a form element
+     */
+    removeCustomDisplayErrorElement(name) {
+        delete this.errorDisplayElements[name];
+    }
+
+    /**
+     * Method for removing elements where the errors related to the form elements with the passed names should be displayed
+     * @param {Array<string>} names - Array with names related to the form elements
+     */
+    removeCustomDisplayErrorElements(names) {
+        if (!(names instanceof Array)) return;
+
+        for (const name of names) {
+            if (typeof name !== 'string') continue;
+            if (this.errorDisplayElements[name] === undefined) continue;
+            delete this.errorDisplayElements[name];
+        }
+    }
+
+    /**
+     * Will set user defined custom validators about form element with the passed name
+     * @param {string} name - The name attribute value of a form element 
+     * @param {{[errorType: string]: {method?: Function<boolean>, errorMessage?: (element:HTMLElement) => string}}} validators - Custom validators configuration
+     */
+    setCustomValidators(name, validators) {
+        if (!name || !validators) return;
+        if (typeof validators !== 'object') return;
+        this.customValidators[name] = validators;
+    }
+
+    /**
+     * Method for removing custom validator
+     * @param {string} name - The name attribute value of a form element
+     */
+    removeCustomValidator(name) {
+        delete this.customValidators[name];
```

Comment:
Check if not a string and display error.

Reply:
fixed